docs/dev-setup.md

```diff
@@ -125,7 +125,7 @@ Override the default gradle settings if you want to run your tests on a differen
 
 ### Server side tests
 1. Start with a clean test-database when running tests: `./gradlew -PtestEnv dbDrop dbMigrate dbLoad`
-1. Make sure the Keycloak authentication server is started (in a Docker container) in your local development environment: `./gradlew dockerCreateKeycloak dockerStartKeycloak`
+1. Make sure the Keycloak authentication server is started (in a Docker container) in your local development environment: `./gradlew dockerCreateKeycloak dockerConfigureKeycloak`
```

Comment:
You still need to start it: :smile: 
1. Make sure the Keycloak authentication server is started (in a Docker container) in your local development environment: `./gradlew dockerCreateKeycloak dockerConfigureKeycloak dockerStartKeycloak`

Also, don't forget to update line 155 of `.github/workflows/build-gradle.yml`:
```
    - run: ./gradlew -PtestEnv dockerCreateKeycloak dockerStartKeycloak
```
to:
```
    - run: ./gradlew -PtestEnv dockerCreateKeycloak dockerConfigureKeycloak dockerStartKeycloak
```